src/filesystem/impls/filer/lib/CSSRewriter.js

```diff
@@ -28,6 +28,7 @@ define(function (require, exports, module) {
         function aggregate(content, callback) {
             var urls = [];
             var urlRegex = new RegExp('url\\([\\\'\\"]?([^\\\'\\"\\)]+)[\\\'\\"]?\\)', 'g');
+            var urlQuotesRegex = new RegExp(/[\\\'\\"]([^\\\'\\"\\)]+)[\\\'\\"]/, 'g');
```

Comment:
I'm a little confused why this is all `new RegExp` when assignment based on regexp literal is perfectly fine in JS, and is a lot easier to read. Especially if we want to keep maintaining this, I would strongly advocate changing the regexp in this PR so that we have the following block instead:
```
var urlRegex = /url\(['"]?([^'"\)]+)['"]?\)/g;
var urlQuotesRegex = /['"]([^'"\)]+)['"]/g;
var periodRegex = /\./g;
var forwardSlashRegex = /\//g;
var dashRegex = /-/g;
...
```
(I'll be honest: I don't understand why some of these regexp even exist)

With all that said, if the intention is to do something smart with imports, rather than "urls" in general (which you'll hit in things like `@font-face { src }` or in properties like `background-image` etc, then we should regexp specifically for imports:

```
var importRule = /@import\s+([^;]+)\r?\n/g;
```
and then do the replacement specifically to the import arguments, with a function that can chunk up the argument string and do the right thing to the right parts:
```
content = content.replace(importRule, function(fullPattern, captured) {
  if (!captured) {
    // this case is kind of an soft-error, really
    return;
  }
  checkImportForURL(captured);
});
```
with a utility function:
```
checkImportForURL((importArgument) {
  var arguments = importArgument.split(' ');
  if (arguments.length === 1) {
    ...
    // check what the arg was based on what we know about the import syntax,
    // extract the URL, and add it to the list of urls to rewrite.
  }
  ... // etc.
}
```